layouts/shortcodes/djm-install-troubleshooting.en.md

```diff
@@ -0,0 +1,10 @@
+The init script installs the Datadog Agent. To make sure it is properly installed, run the Agent status command:
```

Comment:
Can you add details of where and how you can run this? e.g. could just be links to how to ssh into a node on the cluster for the different platforms.

Reply:
Just a heads up, we don't use the FAQ style at Datadog. So let's keep this as a troubleshooting section.

Reply:
Good to know, thanks a lot !